bucket/touchcursor.json

```diff
@@ -0,0 +1,23 @@
+{
+    "version": "1.7.1",
+    "description": "TouchCursor lets you use the home keys as cursor keys – in all Windows programs – keeping your fingers in the best position for fast typing.",
+    "homepage": "https://martin-stone.github.io/touchcursor/",
+    "license": "GPL-3.0-only",
+    "url": "https://github.com/martin-stone/touchcursor/releases/download/v1.7.1/TouchCursor-1.7.1.zip",
+    "hash": "3c4a4361d8f32f3b594a3021c5159d1efe670310126ce99ed8d07d4e8b9c4e77",
+    "extract_dir": "TouchCursor",
+    "bin": "touchcursor.exe",
+    "shortcuts": [
+        [
+            "touchcursor.exe",
+            "touchcursor"
+        ]
+    ],
+    "checkver": {
+        "url": "https://martin-stone.github.io/touchcursor/",
```

Comment:
Why not checkver: github?

Reply:
I can certainly do that if you prefer the manifest to parse github releases.

Reply:
Actually we can just remove `checkver` and `autoupdate`. The software has not been updated for a few years, so we don't need to check it. This saves resources (i.e. number of connections) for out *Excavator* bot.